src/components/10-atoms/spinner/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@axa-ch/spinner",
+  "version": "0.0.0-beta.0",
+  "description": "The spinner component for the AXA Pattern Library",
+  "author": "Pattern Warriors",
+  "homepage": "https://github.com/axa-ch/patterns-library/tree/develop/src/components/10-atoms/spinner#readme",
+  "license": "Copyright 2019 AXA Versicherungen AG",
```

Comment:
Hm... Maybe we should remove the year here? I know, that this will be the same problem in all components.